website/src/redux/reducers/ReducerConstants.jsx

```diff
@@ -56,3 +45,8 @@ export const ELEMENT_ROTATION = 'rotation';
 export const ELEMENT_MODULE = 'module';
 export const ELEMENT_ATTRS = [ELEMENT_POSITION, ELEMENT_SIZE, ELEMENT_ROTATION,
   ELEMENT_MODULE]
+
+// Fields for canvas.
+export const CANVAS_NAME = 'name';
+export const CANVAS_OWNER = 'owner';
+export const CANVAS_USERS = 'users';
```

Comment:
This will be for a specific canvas in the canvas list so not in the blank state.